<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.13">
  <compounddef id="classboost_1_1geometry_1_1invalid__output__exception" kind="class" language="C++" prot="public">
    <compoundname>boost::geometry::invalid_output_exception</compoundname>
    <basecompoundref refid="classboost_1_1geometry_1_1exception" prot="public" virt="non-virtual">boost::geometry::exception</basecompoundref>
    <includes local="no">exception.hpp</includes>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classboost_1_1geometry_1_1invalid__output__exception_1ae376a7f298b0abd466f1c6f1768ab1c2" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>boost::geometry::invalid_output_exception::invalid_output_exception</definition>
        <argsstring>()</argsstring>
        <name>invalid_output_exception</name>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/home/hsb/CProj/Learning/boost/boost_1_79_0/boost/geometry/core/exception.hpp" line="97" column="1" bodyfile="/home/hsb/CProj/Learning/boost/boost_1_79_0/boost/geometry/core/exception.hpp" bodystart="97" bodyend="97"/>
      </memberdef>
      <memberdef kind="function" id="classboost_1_1geometry_1_1invalid__output__exception_1af62848f377cc439d8634906a37f05fcc" prot="public" static="no" const="yes" explicit="no" inline="yes" virt="virtual">
        <type>char const  *</type>
        <definition>virtual char const* boost::geometry::invalid_output_exception::what</definition>
        <argsstring>() const</argsstring>
        <name>what</name>
        <reimplements refid="classboost_1_1geometry_1_1exception_1a20b8de7e5200fba44f60e45345377113">what</reimplements>
        <exceptions> throw ()</exceptions>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/home/hsb/CProj/Learning/boost/boost_1_79_0/boost/geometry/core/exception.hpp" line="99" column="1" bodyfile="/home/hsb/CProj/Learning/boost/boost_1_79_0/boost/geometry/core/exception.hpp" bodystart="99" bodyend="102"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
<para>Invalid Output Exception. </para>    </briefdescription>
    <detaileddescription>
<para>The <ref refid="classboost_1_1geometry_1_1invalid__output__exception" kindref="compound">invalid_output_exception</ref> is thrown if valid output cannot be generated by a function even if arguments are valid, e.g. union of geographic polygons covering more than half of the area of the globe. </para>    </detaileddescription>
    <inheritancegraph>
      <node id="608">
        <label>boost::geometry::exception</label>
        <link refid="classboost_1_1geometry_1_1exception"/>
        <childnode refid="609" relation="public-inheritance">
        </childnode>
      </node>
      <node id="607">
        <label>boost::geometry::invalid_output_exception</label>
        <link refid="classboost_1_1geometry_1_1invalid__output__exception"/>
        <childnode refid="608" relation="public-inheritance">
        </childnode>
      </node>
      <node id="609">
        <label>std::exception</label>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="611">
        <label>boost::geometry::exception</label>
        <link refid="classboost_1_1geometry_1_1exception"/>
        <childnode refid="612" relation="public-inheritance">
        </childnode>
      </node>
      <node id="610">
        <label>boost::geometry::invalid_output_exception</label>
        <link refid="classboost_1_1geometry_1_1invalid__output__exception"/>
        <childnode refid="611" relation="public-inheritance">
        </childnode>
      </node>
      <node id="612">
        <label>std::exception</label>
      </node>
    </collaborationgraph>
    <location file="/home/hsb/CProj/Learning/boost/boost_1_79_0/boost/geometry/core/exception.hpp" line="94" column="1" bodyfile="/home/hsb/CProj/Learning/boost/boost_1_79_0/boost/geometry/core/exception.hpp" bodystart="93" bodyend="103"/>
    <listofallmembers>
      <member refid="classboost_1_1geometry_1_1invalid__output__exception_1ae376a7f298b0abd466f1c6f1768ab1c2" prot="public" virt="non-virtual"><scope>boost::geometry::invalid_output_exception</scope><name>invalid_output_exception</name></member>
      <member refid="classboost_1_1geometry_1_1invalid__output__exception_1af62848f377cc439d8634906a37f05fcc" prot="public" virt="virtual"><scope>boost::geometry::invalid_output_exception</scope><name>what</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
